packages/movit/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://movit.sesse.net/
+TERMUX_PKG_DESCRIPTION="The modern video toolkit. "
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_VERSION=1.7.1
+TERMUX_PKG_SRCURL=https://movit.sesse.net/movit-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=b33073b705f0ccb6ac4942cf51151515407b40bb4e9a2dd0228c1c2cb1fbc11a
+TERMUX_PKG_DEPENDS="fftw, libepoxy"
+TERMUX_PKG_BUILD_DEPENDS="eigen, googletest, sdl2"
+TERMUX_PKG_SUGGESTS="$TERMUX_PKG_BUILD_DEPENDS"
```

Comment:
Why would the finished package recommend its build time dependencies?

Reply:
Also I just noticed, `sdl2` is in the x11 repo, so this package would also need to be in the x11 section.

Reply:
Should be a mistake of mine when copying from `mlt` build script (which other linux distros usually use similar lists of build deps and optional deps)